eng/build.yml

```diff
@@ -64,6 +66,7 @@ jobs:
 
     variables: 
     - ${{ insert }}: ${{ parameters.variables }}
+    - _BuildConfig: ${{ parameters.configuration }}
```

Comment:
I opened https://github.com/dotnet/arcade/issues/6972 because this shouldn't be necessary.